roles/splunk_monitor/tasks/initialize_standalone_search_head.yml

```diff
@@ -0,0 +1,43 @@
+---
+  - name: Set default captaincy
+    set_fact:
+      splunk_search_head_captain: false
+
+  - name: Determine captaincy
+    set_fact:
+      splunk_search_head_captain: true
+    when: ansible_hostname == splunk.search_head_captain_url or
+          ansible_fqdn == splunk.search_head_captain_url or
+          splunk.hostname == splunk.search_head_captain_url or
+          splunk.role == "splunk_search_head_captain"
+
+  - include_tasks: ../../../roles/splunk_monitor/tasks/adding_peers.yml
+
+  - include_tasks: ../../../roles/splunk_common/tasks/set_as_deployment_client.yml
+    when:
+      - splunk.deployment_server is defined
+      - splunk.deployment_server
+      - not splunk_search_head_cluster | bool
+
+  - include_tasks: ../../../roles/splunk_common/tasks/provision_apps.yml
+    when:
+      - splunk.apps_location
+      - not splunk_search_head_cluster | bool
+      - splunk.deployment_server is not defined or not splunk.deployment_server
+
+  - include_tasks: setup_multisite.yml
+    when:
+      - splunk.site is defined
+      - splunk.multisite_master is defined
+
+  - include_tasks: ../../splunk_common/tasks/enable_forwarder_monitoring.yml
+    when:
+      - dmc_forwarder_monitoring is defined
+      - dmc_forwarder_monitoring | bool
+
+  - include_tasks: ../../../roles/splunk_common/tasks/enable_dfs.yml
```

Comment:
DFS is another feature that we don't need to support on the DMC role, so I think it's probably ok to remove this